app/views/landmarks/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h3><%= @landmark.name %></h3>
-<%= @landmark.user.email %>
```

Comment:
@mlg- We aren't adding any styling until we get the core functionality hammer out